projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts

```diff
@@ -0,0 +1,81 @@
+import { newE2EPage, E2EElement, E2EPage } from '@stencil/core/testing';
+import { EventSpy } from '@stencil/core/dist/declarations';
+
+describe('dot-select', () => {
+
+    let page: E2EPage;
+    let element: E2EElement;
+    let spyStatusChangeEvent: EventSpy;
+    let spyValueChangeEvent: EventSpy;
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+
+        await page.setContent(`
+        <dot-select
+            name="testName"
+            label="testLabel"
+            hint="testHint"
+            options="|,valueA|1,valueB|2"
```

Comment:
If we have spaces in the values or special chars did it work?

Reply:
yes